pybigquery/sqlalchemy_bigquery.py

```diff
@@ -99,6 +99,7 @@ def format_label(self, label, name=None):
     'BYTES': types.BINARY,
     'TIME': types.TIME,
     'RECORD': types.JSON,
+    'NUMERIC': types.Numeric,
```

Comment:
Looks like Numeric is a base class. We might prefer `sqlalchemy.types.DECIMAL` instead.

Reply:
Yes, good point. Done.